utils/src/com/goide/GoEnvironmentUtil.java

```diff
@@ -47,4 +49,19 @@ public static String retrieveGoPathFromEnvironment() {
     String path = EnvironmentUtil.getValue(GoConstants.GO_PATH);
     return path != null ? path : PathMacros.getInstance().getValue(GoConstants.GO_PATH);
   }
+  
+  @NotNull
+  public static String[] getEnvironment(@NotNull Project project, @NotNull String sdkHomePath) {
+    String[] env = EnvironmentUtil.getEnvironment();
+    
+    for (int i=0; i < env.length; i++) {
+      if (env[i].toLowerCase().startsWith("path=")) {
```

Comment:
> What is it? Why do I need this?
This is needed as without it, on my system, IDEA can't process the correct path. When running ```go fmt``` that will execute ```gofmt``` but since it's calling the executable without a full path, as the ```PATH``` environment variable is not properly passed, this will fail.

There's also a problem that I've commented on [YouTrack](https://youtrack.jetbrains.com/issue/IDEA-118483). Also, from the previous experience with 0.9.x anything that relies on the environment variables is not happily handled in IDEA and it's better to stay away from it.